src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/AssetDefinitions/AssetDefinition.cs

```diff
@@ -0,0 +1,29 @@
+﻿using EasyAbp.BookingService.AssetCategories;
+using EasyAbp.BookingService.AssetSchedules;
+using JetBrains.Annotations;
+
+namespace EasyAbp.BookingService.AssetDefinitions;
+
+public class AssetDefinition
+{
+    [NotNull]
+    public string Name { get; }
+    
+    /// <summary>
+    /// This property determines whether assets can be occupied by default when there is no schedule created.
+    /// The property value from <see cref="AssetCategory"/> is preferred.
+    /// </summary>
+    public virtual AssetSchedulePolicy DefaultSchedulePolicy { get; }
+    
+    /// <summary>
+    /// How many days in advance can you occupy the asset.
+    /// The property value from <see cref="AssetCategory"/> is preferred.
+    /// </summary>
+    public virtual int DaysInAdvance { get; protected set; }
```

Comment:
`protected set`  needed?

Reply:
https://github.com/abpframework/abp/blob/dev/modules/blogging/src/Volo.Blogging.Domain/Volo/Blogging/Comments/Comment.cs#L10-L14

I am not sure whether a property should always have a protected setter, but doing it is safer for now.

Reply:
Sorry, I was wrong. This `AssetDefinition` is not an entity class. The setter can be removed.